cirq/contrib/noise_models/noise_models.py

```diff
@@ -211,54 +212,142 @@ def noisy_moment(self, moment: 'cirq.Moment',
             ]
 
 
-class PerQubitDepolarizingNoiseModel(devices.NoiseModel):
-    """DepolarizingNoiseModel which allows depolarization probabilities to be
-    specified separately for each qubit.
+class PerQubitDepolarizingWithDampedReadoutNoiseModel(devices.NoiseModel):
+    """DepolarizingWithDampedReadoutNoiseModel which allows depolarization
+    probabilities to be specified separately for each qubit.
 
-    Similar to depol_prob in DepolarizingNoiseModel, depol_prob_map should map
-    Qids in the device to their depolarization probability.
+    This simulates asymmetric readout error. The noise is structured
+    so the T1 decay is applied, then the readout bitflip, then measurement.
```

Comment:
Wording is slightly confusing.  Should be something more like
> T1 decay is applied after all gates that are not measurements, then for moments that are all measurements amplitude damping followed by a bit flip channel is applied before the measurement.  

I would also mention that this noise channel only works when all measurements are alone in moments.

Reply:
Done.